script/upgrades/MU03/MU03.sol

```diff
@@ -557,6 +558,18 @@ contract MU03 is IMentoUpgrade, GovernanceScript {
     );
   }
 
+  function proposal_whitelistMissingOracleProvider(MU03Config.MU03 memory config) public {
+    // Add DiWu as oracle provider for the EUROC/EUR rate feed
+    address diwuOracleAddress = 0xBD136a625299A0ac5Ca7Ce9220aCA6e08a624e37;
```

Comment:
where can I verify this ?

Reply:
This is the message where he shared his address https://discord.com/channels/966739027782955068/1044219395553120276/1142131663397208094